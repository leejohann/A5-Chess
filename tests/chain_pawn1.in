move 0 6 0 5
move 7 1 7 2
move 0 5 0 4
move 7 2 7 3
move 0 4 0 3
move 7 3 7 4
move 0 3 0 2
move 7 4 7 5
quit
