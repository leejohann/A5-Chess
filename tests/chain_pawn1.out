RNBQKBNR
PPPPPPPP
 _ _ _ _
_ _ _ _ 
 _ _ _ _
_ _ _ _ 
pppppppp
rnbqkbnr

it is White's turn
RNBQKBNR
PPPPPPPP
 _ _ _ _
_ _ _ _ 
 _ _ _ _
p _ _ _ 
 ppppppp
rnbqkbnr

it is Black's turn
RNBQKBNR
PPPPPPP 
 _ _ _ P
_ _ _ _ 
 _ _ _ _
p _ _ _ 
 ppppppp
rnbqkbnr

it is White's turn
RNBQKBNR
PPPPPPP 
 _ _ _ P
_ _ _ _ 
p_ _ _ _
_ _ _ _ 
 ppppppp
rnbqkbnr

it is Black's turn
RNBQKBNR
PPPPPPP 
 _ _ _ _
_ _ _ _P
p_ _ _ _
_ _ _ _ 
 ppppppp
rnbqkbnr

it is White's turn
RNBQKBNR
PPPPPPP 
 _ _ _ _
p _ _ _P
 _ _ _ _
_ _ _ _ 
 ppppppp
rnbqkbnr

it is Black's turn
RNBQKBNR
PPPPPPP 
 _ _ _ _
p _ _ _ 
 _ _ _ P
_ _ _ _ 
 ppppppp
rnbqkbnr

it is White's turn
RNBQKBNR
PPPPPPP 
p_ _ _ _
_ _ _ _ 
 _ _ _ P
_ _ _ _ 
 ppppppp
rnbqkbnr

it is Black's turn
RNBQKBNR
PPPPPPP 
p_ _ _ _
_ _ _ _ 
 _ _ _ _
_ _ _ _P
 ppppppp
rnbqkbnr

it is White's turn

